src/lib/transform-node-with-custom-properties.js

```diff
@@ -76,8 +76,5 @@ function asClonedArray(array, parent) {
 
 // whether the node is a var() function
 function isVarFunction(node) {
-	return Object(node).type === 'func' && varRegExp.test(node.value) && Object(node.nodes).length > 0;
+	return Object(node).name === 'var' && Object(node.nodes).length > 0;
```

Comment:
Could the type still be tested, and the name still be tested using a regular expression? It would seem like it could based upon https://github.com/shellscape/postcss-values-parser/blob/master/docs/Func.md

```suggestion
	return Object(node).type === 'func' && varRegExp.test(node.name) && Object(node.nodes).length > 0;
```

Reply:
Sure! Just pushed a commit that brings the regex back with all tests still passing